test/isochrone.cc

```diff
@@ -187,40 +187,45 @@ TEST(Isochrones, Basic) {
   GraphReader reader(cfg.get_child("mjolnir"));
 
   {
+    SCOPED_TRACE("basic request 1 failed");
```

Comment:
this is golden for lots of our test files where we so often have helper functions do the actual assertions and we often don't know which test actually failed.